pkg/shell/machines.js

```diff
@@ -584,6 +584,17 @@ define([
         "#024848"
     ];
 
+    module.colors.parse = function parse_color(input) {
+        var div = document.createElement('div');
+        div.style.color = input;
+        var style = window.getComputedStyle(div, null);
+        for (var i = 0; i < style.length; i++) {
+            if (style[i] == "color")
+                return style.getPropertyValue(style[i]);
+        }
+        return div.style.color;
```

Comment:
I was just wondering and haven't read up on it other than [1], which says that every browser (IE since 9) supports it.

[1] https://developer.mozilla.org/en-US/docs/Web/API/Window/getComputedStyle